### PR TITLE
Made brightcoveVideo work with react-native link

### PR DESCRIPTION
**Pull request by adrienthiery**
I ran tests, everything seems fine and I integrated the package in my current project and everything went smoothly.

Any feedback is welcome 😄

### CODE REVIEW

**Comment by trevorah** [comment]
@adrienthiery Thank you for the PR!

The tests are failing because of `yarn lint`. The new peerDependencies will need to be in devDependencies too.

**Comment by adrienthiery** [comment]
Hi @trevorah ,

So, I'm not sure what should be a `peerDependency` and what should be a `devDependency`... 

In my honest opinion, I guess `react-dom`, `react-native` and  `react-native-web` are `devDependencies` (`react-dom` being used in the test file), but `react` being a dependency of both `react-native` and `react-native-web`, I would keep it as a `peerDependency` only...

But I'm not fully sure about all that 😅 

What do you think?

**Comment by adrienthiery** [comment]
Fixes #106

**Comment by RussellToon** [comment]
Hi, It would be nice to have this as a framework (or CocoaPod), but I don't think this achieves that, as it doesn't build an archive due to missing dependencies on React and Brightcove. We will try to resolve this at some point, but I don't think this project file currently provides any more utility than "dragging and dropping" the directory into your project, right?

File: packages/brightcove-video/ios/RNTBrightcove.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,302 @@
+// !$*UTF8*$!
```
Comment (RussellToon) [inline packages/brightcove-video/ios/RNTBrightcove.xcodeproj/project.pbxproj]: I.e. this file

**Comment by adrienthiery** [comment]
Hi,
Well, it actually removes the step you had before to drag n drop the actual files into your project by making it work with `react-native link` and the only step left to do for using the package is then to add the Brightcove SDK with cocoapods.

From my perspective, that's a valuable improvement.

**Pull request merged**